product/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path, include
+from .views      import CategoryView, CategoryDetailView
+
+urlpatterns = [
+        path('/category', CategoryView.as_view()),
+        path('/category/<int:Category_id>', CategoryDetailView.as_view())
+
+]
```

Comment:
- whitespace 관리 신경써주세요 😁
- 그리고 path param 활용 잘하셨습니다 👍
```suggestion
urlpatterns = [
    path('/category', CategoryView.as_view()),
    path('/category/<int:Category_id>', CategoryDetailView.as_view())
]
```

Reply:
수정 완료했습니다.!! 감사합니다.!